src/components/common/Chart/Chart.tsx

```diff
@@ -27,6 +38,7 @@ export interface ChartProps {
   chartId: string;
   chartLabels?: string[];
   chartValues?: number[];
+  chartData?: ChartData;
```

Comment:
Yeah, I thought to keep them both, to have a simpler option, but I think it clutters the interface. I'll remove it

Reply:
Will leave it to you. Theoretically will be good to have just one way of supplying the data - otherwise gets ambiguous. Whatever makes sense practically is totally fine.